src/Avalonia.Themes.Fluent/Controls/SplitButton.xaml

```diff
@@ -0,0 +1,205 @@
+﻿<Styles xmlns="https://github.com/avaloniaui"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        x:CompileBindings="True">
+
+  <Design.PreviewWith>
+    <Border Padding="60">
+      <StackPanel>
+        <SplitButton Content="Hello World">
+          <SplitButton.Flyout>
+            <Flyout>Hello</Flyout>
+          </SplitButton.Flyout>
+        </SplitButton>
+        <ToggleSplitButton Content="Hello World">
+          <ToggleSplitButton.Flyout>
+            <Flyout>Hello</Flyout>
+          </ToggleSplitButton.Flyout>
+        </ToggleSplitButton>
+      </StackPanel>
+    </Border>
+  </Design.PreviewWith>
+
+  <Styles.Resources>
+    <x:Double x:Key="SplitButtonPrimaryButtonSize">32</x:Double>
+    <x:Double x:Key="SplitButtonSecondaryButtonSize">32</x:Double>
+    <StreamGeometry x:Key="SplitButtonSecondaryButtonIconGeometry">M1939 486L2029 576L1024 1581L19 576L109 486L1024 1401L1939 486Z</StreamGeometry>
+  </Styles.Resources>
+
+  <Style Selector="SplitButton">
```

Comment:
Comment was made here: https://github.com/AvaloniaUI/Avalonia/pull/7422#issuecomment-1021911616

> ToggleSplitButton must have separated style. Otherwise, it will be pain to restyle it independently (you can't use ToggleSplitButton in selector because of StyleKey).

 * Keeping ToggleSplitButton and SplitButton together is a good idea for simplification -- it was an elegant idea from WinUI. There is established precedent and real-world usage/discussion.
 * I still think ToggleSplitButton can be styled separately to some degree using classes if that is ever needed.
 * It is of course still possible to modify the shared default. In how many cases would someone need to change one but not the other?

Overall, this is something I would keep in mind but get real-world feedback on. I would be willing to bet anyone customizing ToggleSplitButton is also going to need to customize SplitButton and keeping them the same style will save them time along with us.

If after some time it becomes obvious in real-world code that it was a bad idea to combine them, it can be changed. Until then creating 4 style variants clearly seems like an unnecessary maintenance and development burden.

Reply:
It's not consistent with current Button/ToggleButton styles, but I hear you. We can leave it as is for now.

Reply:
Although it's possible to reuse all of these styles by extending selectors to match both controls.

1. You can duplicate selectors of the same style in a way from:
```
SplitButton:checked /template/ Button#PART_PrimaryButton:pressed /template/ ContentPresenter
```
to
```
SplitButton:checked /template/ Button#PART_PrimaryButton:pressed /template/ ContentPresenter,
ToggleSplitButton:checked /template/ Button#PART_PrimaryButton:pressed /template/ ContentPresenter
```
But I am not personally fan of it, since it makes fat selectors hard to read. Not to mention it will be even worse for selectors where we already have it combined for both internal buttons.
Btw it's yet another place that can be improved with inner styles.

2. You can replace "SplitButton:checked" with ":is(SplitButton):checked", which will work for both types, without need to define StyleKey. 
3. Or we can keep it as it is for now.

Reply:
> It's not consistent with current Button/ToggleButton styles,

Yes, these are a bit simpler -- ToggleButton is still a big long though but without the fancy selector work.

> We can leave it as is for now.

Marking resolved as we agree.

There are even more options to reuse the styles as you mentioned above (with some additional selector work). So this can be revisited if some good real-world examples come in the future to justify more work here. At that point I would prefer going the selector route and if that still isn't ideal, duplicate the styles. I'm still betting that doesn't happen.